container_config/orderlyweb/config.properties

```diff
@@ -1 +1,2 @@
-montagu.api_url=http://montagu_api_1:8080/v1
\ No newline at end of file
+montagu.api_url=http://montagu_api_1:8080/v1
```

Comment:
commenting here so that it's nearby, but why are the go_signal files included? You are using `touch` to create them in the startup. I feel like I am not seeing something in the setup or testing here where these sneak across, and that could do with a comment if it's the case

Reply:
Yeah, this is actually just used in the test script I think but could probably do without it and use touch as elsewhere, will put that in. :+1: